viewsets/templates/forms/inline.html

```diff
@@ -22,5 +22,5 @@
             {% endfor %}
         </div>
     </fieldset>
-    <button class="btn btn-default add-more"><span class="glyphicon glyphicon-plus"></span></button>
```

Comment:
this is the example from the [bootstrap site](http://getbootstrap.com/components/):

```
<button type="button" class="btn btn-default" aria-label="Left Align">
  <span class="glyphicon glyphicon-align-left" aria-hidden="true"></span>
</button>
```